client/src/graphs/templates/useTemplate.ts

```diff
@@ -0,0 +1,96 @@
+import type { Graph } from "@graph/types";
+import type { GraphTemplate } from "./types";
+import { computed, ref } from "vue";
+import {
+  centerNodesOnOriginCoordinates,
+  getAverageCoordinatesOfGraphNodes,
+  getGraphNodesBoundingBox,
+} from "./helpers";
+import { generateId } from "@utils/id";
+import { useLocalStorage } from "@vueuse/core";
+import { createImageFromCanvasRegion } from "./snapshot";
+import { products } from "@utils/product";
+
+export const useTemplate = (graph: Graph) => {
+  const userTemplates = useLocalStorage<GraphTemplate[]>(
+    "user-graph-templates",
```

Comment:
i dont think you need to prepend this with "user-" if its going into local storage